projection/core/src/main/scala/com/lightbend/lagom/internal/projection/WorkerHolderActor.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.internal.projection
+
+import akka.actor.Actor
+import akka.actor.ActorLogging
+import akka.actor.ActorRef
+import akka.actor.Props
+import akka.actor.Terminated
+import akka.pattern.BackoffOpts
+import akka.pattern.BackoffSupervisor
+import com.lightbend.lagom.internal.cluster.ClusterDistribution.EnsureActive
+import com.lightbend.lagom.internal.projection.ProjectionRegistryActor.WorkerCoordinates
+import com.lightbend.lagom.projection.Started
+import com.lightbend.lagom.projection.Status
+import com.lightbend.lagom.projection.Stopped
+
+import scala.concurrent.duration._
+
+object WorkerHolderActor {
```

Comment:
I would like to find a better name to this.

Some suggestions:
* WorkerParent (why not?)
* WorkerDriver
* Worker
* Partition

I don't remember the reason why we introduced a parent. Maybe because we are in need to fit the current design (read-side processor / topic producer in it).

Reply:
`WorkerParent` and `WorkerCoordinator` are better names, in my opinion.

Reply:
Hmmm, `SomethingPartition` or `PartitionCoordinator` or similar sounds good to me too. Makes it very clear it takes care of a single partition.